### PR TITLE
Update Linkerd 2.x in landscape with updated info

### PR DESCRIPTION
**Pull request by metsuke0**
https://github.com/layer5io/layer5/issues/372

### CODE REVIEW

**Comment by leecalcote** [comment]
Yes! Thank you, @metsuke0! 🎉 

A minor item to attend to, if you would - [sign your commit](https://github.com/layer5io/layer5/pull/373/checks?check_run_id=531689827) in this PR.

Here are a couple methods to make signing commits a non-issue (done automatically) in the future - https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin

**Comment by leecalcote** [comment]
@hawkw @grampelberg a quick review from you here would be great.

File: _data/landscape/meshes.yml [hunk 1]
```diff
@@ -157,10 +157,10 @@
   h2: "Yes"
   grpc: "Yes"
   tcp-web: "Yes"
-  multi-cluster: "?"
+  multi-cluster: "Experimental"
   multi-tenant: "?"
```
Comment (grampelberg) [inline _data/landscape/meshes.yml]: We're multi-tenant as all write operations are controlled by k8s RBAC.

File: _data/landscape/meshes.yml [hunk 2]
```diff
@@ -157,10 +157,10 @@
...
   prometheus: "Yes"
-  tracing: "None"
+  tracing: "Jaeger"
```
Comment (grampelberg) [inline _data/landscape/meshes.yml]: Technically this is *all* as we're integrated with OC/OT.

**Comment by ashiscs** [comment]
fixes #372

**Pull request merged**